lib/Conversion/SeqToSV/SeqToSV.cpp

```diff
@@ -448,26 +423,58 @@ void SeqToSVPass::runOnOperation() {
   // Lower memories and registers in modules in parallel.
   bool needsRegRandomization = false;
   bool needsMemRandomization = false;
+  DenseMap<HWModuleOp, std::pair<bool, bool>> moduleFragments;
```

Comment:
The `std::pair<bool, bool>` makes it difficult to know what means what. A `struct` would be better.